projects/aca-content/src/lib/aca-content.module.ts

```diff
@@ -97,7 +99,10 @@ import { SearchResultsRowComponent } from './components/search/search-results-ro
     AcaFolderRulesModule,
     CreateFromTemplateDialogComponent,
     OpenInAppComponent,
-    UploadFilesDialogComponent
+    UploadFilesDialogComponent,
+    LottieModule.forRoot({
```

Comment:
this might cause lazy loading problems, as 'aca-content' is a library module, not an application one

Reply:
I have changed it a bit now

Reply:
removed lottie library